RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHBHE_cfi.py

```diff
@@ -31,3 +42,64 @@
 
 )
 
+cuts2017 = particleFlowRecHitHBHE.producers[0].qualityTests[0].cuts
```

Comment:
considering copy-pastes everywhere, it makes more sense to me to have a definition of cuts for an individual detector variant 
```python
# defaults to run-1
_thresholdsHB = cms.vdouble(0.8, 0.8, 0.8, 0.8)
_thresholdsHE = cms.vdouble(0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8)
_thresholdsHBphase1 = cms.vdouble(0.1, 0.2, 0.3, 0.3)
_thresholdsHEphase1 = cms.vdouble(0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2)
run2_HCAL_2018.toReplaceWith(_thresholdsHE, _thresholdsHEphase1)
... 
 particleFlowRecHitHBHE = cms.EDProducer("PFRecHitProducer",
... 
                  cuts = cms.VPSet(
                        cms.PSet(
                            depth=cms.vint32(1, 2, 3, 4),
                            threshold = _thresholdsHB,
                            detectorEnum = cms.int32(1)
                            ),
                        cms.PSet(
                            depth=cms.vint32(1, 2, 3, 4, 5, 6, 7),
                            threshold = _thresholdsHE,
                            detectorEnum = cms.int32(2)

```

this way correct values are created from the start and there is no need to make rather dangerous modifications of VPSets based on just indices ```producers[0].qualityTests[0]```

Reply:
actually, these same vectors will be useful in particleFlowClusterHBHE_cfi.py.
So, it may be better to even make a new file particleFlowHitThresholdsHBHE_cfi.py and define in them all values reused in multiple places e.g. ```pfHitThresholdsHB = cms.vdouble(0.8, 0.8, 0.8, 0.8)```

Reply:
I agree with this redefinition

Reply:
toReplaceWith does not work with type <class 'FWCore.ParameterSet.Types.vdouble'>

Reply:
err. I guess I need to try out my suggestions.
I think that in this case we can refactor the python configuration later.
Please focus on having a working version first.